cmd/supervisor/supervisor.go

```diff
@@ -0,0 +1,16 @@
+// (c) Cartesi and individual authors (see AUTHORS)
```

Comment:
Let's rename this package. I don't want to give the wrong impression with the supervisor name. Probably, it is best to stick with the name of the final binary. Something like `rollups-node`, `cartesi-node`, or `cartesi-rollups-node`.